ext/src/main/java/com/kickstarter/jruby/Telekinesis.java

```diff
@@ -33,10 +34,12 @@
 public class Telekinesis {
     /**
      * Create a new KCL {@link Worker} that processes records using the given
-     * {@link ExecutorService} and {@link IRecordProcessorFactory}.
+     * {@link ExecutorService} and {@link IRecordProcessorFactory}. Uses the
+     * given {@link AmazonDynamoDBClient} if non-null.
      */
     public static Worker newWorker(final KinesisClientLibConfiguration config,
                                    final ExecutorService executor,
+                                   final AmazonDynamoDBClient dynamoClient,
```

Comment:
I generally like to stick to the interface type where possible. can you make this an `AmazonDynamoDB` instead?